### PR TITLE
Add example attributes on documentation

### PR DESCRIPTION
**Pull request by maxhelias**
I'm not sure if i use the right `configuration-bloc`, maybe @javiereguiluz can review it, please ?

### CODE REVIEW

File: doc/providers/totp.rst [hunk 1]
```diff
@@ -57,43 +57,80 @@ and the period of the temporary codes.
     Be warned, custom configurations will not be compatible with the defaults of Google Authenticator app any more. You
     will have to use another application (e.g. FreeOTP on Android).
 
-.. code-block:: php
-
-   <?php
+.. configuration-block::
+    .. code-block:: php
```
Comment (javiereguiluz) [inline doc/providers/totp.rst:62]: Here, let's add a blank line between `configuration-block` and `clode-block` directives.

**Review by javiereguiluz — COMMENTED** [review]
Yes, I think this is correct. The required RST directive structure is:

```rst
.. configuration-block::

    .. code-block:: php-annotations

        // code here...

    .. code-block:: php-attributes

        // code here...
```

File: doc/providers/totp.rst (hunk 1 above)
Comment (maxhelias) [reply doc/providers/totp.rst:62]: Done! Thanks for the feedback

**Comment by scheb** [comment]
Stupid question: What is this doing compared to the previous implementation?

**Comment by javiereguiluz** [comment]
@scheb it creates a clickable set of tabs where readers can select their preferred format (and that format is persisted on the site):

**Comment by scheb** [comment]
Ok, got it 👍 Sweet!

**Pull request merged**